src/darwin/Framework/CHIP/MTRAsyncWorkQueue.mm

```diff
@@ -160,6 +184,25 @@ - (void)markComplete
     _duplicateCheckHandler = nil;
 }
 
+- (NSString *)description
+{
+    NSString * state;
+    switch (_state) {
+    case MTRAsyncWorkItemMutable:
+        state = @"mutable";
+        break;
+    case MTRAsyncWorkItemComplete:
+        state = @"complete";
+        break;
+    case MTRAsyncWorkItemEnqueued:
+        state = @"enqueued";
+        break;
+    default:
+        return [NSString stringWithFormat:@"<%@ %llu running retry: %tu>", self.class, _uniqueID, self.retryCount];
+    }
+    return [NSString stringWithFormat:@"<%@ %llu %@>", self.class, _uniqueID, state];
```

Comment:
This log and the above also log _uniqueID and should follow the same format "work item [%llu]"

Reply:
This is not a log message and isn’t included in any log messages, so there’s no need to follow the exact format here.